src/Winton.Extensions.Configuration.Consul/ConfigurationBuilderExtensions.cs

```diff
@@ -32,6 +46,22 @@ public static class ConfigurationBuilderExtensions
             return builder.AddConsul(key, cancellationToken, options => { });
         }
 
+        /// <summary>
+        ///     Adds Consul as a configuration source to the <see cref="IConfigurationBuilder" />
+        ///     and applies the given overrides to the <see cref="IConsulConfigurationSource" />.
+        /// </summary>
+        /// <param name="builder">The builder to add consul to</param>
+        /// <param name="key">The key in consul where the configuration is located</param>
+        /// <param name="options">An action used to configure the options of the <see cref="IConsulConfigurationSource" /></param>
+        /// <returns>The builder</returns>
+        public static IConfigurationBuilder AddConsul(
+            this IConfigurationBuilder builder,
+            string key,
+            Action<IConsulConfigurationSource> options)
+        {
+            return builder.AddConsul(key, default, options);
```

Comment:
This causes a build failure on your macOS CI because no LangVersion is specified. Would you rather I changed this to `CancellationToken.None` or specified the `LangVersion` in the `.csproj`?